app/assets/javascripts/demographics_fields.js

```diff
@@ -135,18 +135,30 @@ function applyListenersFor(target) {
     }
   });
 
-  $('select#tribal-state').change(function() {
-    if ($('.featured_tribes_selection').length > 0 && this.value == $('#enroll_state_abbr').val()) {
-      $('.featured-tribe-container').removeClass('hide');
-      var tribe_codes_array = $('.tribe_codes:checked').map(function(){ return $(this).val(); }).get();
-      if (tribe_codes_array.includes("OT")){
-        $('.tribal-name-container').removeClass('hide');
+  // tribal-state change - select from options
+  $('select#tribal-state').on("change", function() {
+    var enroll_state_abbr = $('#enroll_state_abbr').val();
+
+    if (enroll_state_abbr === 'ME') {
```

Comment:
Make use of RR configuration instead of comparing these against the hardcoded text.